openbb_platform/core/openbb_core/app/static/package_builder.py

```diff
@@ -1742,25 +1754,48 @@ def get_paths(cls, route_map: Dict[str, BaseRoute]) -> Dict[str, Dict[str, Any]]
                         provider_parameter_fields = cls._get_provider_parameter_info(
                             standard_model
                         )
-                        reference[path]["parameters"]["standard"].append(
-                            provider_parameter_fields
-                        )
 
                         # Add endpoint data fields for standard provider
                         reference[path]["data"]["standard"] = (
                             cls._get_provider_field_params(standard_model, "Data")
                         )
                         continue
+
                     # Adds provider specific parameter fields to the reference
                     reference[path]["parameters"][provider] = (
                         cls._get_provider_field_params(
                             standard_model, "QueryParams", provider
                         )
                     )
+
                     # Adds provider specific data fields to the reference
                     reference[path]["data"][provider] = cls._get_provider_field_params(
                         standard_model, "Data", provider
                     )
+
+                    # Remove choices from 'standard' if choices for a parameter exist
```

Comment:
if there's standard choices why do we remove it in favor of provider choices? i'd say standard should prevail

Reply:
Because this is a main crux of the issue, combining all choices into one list where there is no distinction between providers. It should always be grouped by provider where multiple  providers exist. "Standard choices" != "valid choices".

Reply:
yap, gotcha